.github/workflows/build.yml

```diff
@@ -34,24 +31,132 @@ jobs:
+        run: sudo apt-get update && sudo apt-get install -y ffmpeg libturbojpeg
+      - name: Install Pillow
+        run: pip install Pillow==6.2.2
+        if: ${{matrix.torchvision == '0.4.2'}}
+      - name: Install PyTorch
+        run: pip install torch==${{matrix.torch}}+cpu torchvision==${{matrix.torchvision}}+cpu -f https://download.pytorch.org/whl/torch_stable.html
+      - name: Install unittest dependencies
+        run: pip install pytest coverage lmdb PyTurboJPEG
       - name: Build and install
         run: rm -rf .eggs && pip install -e .
+      - name: Run unittests
+        run: coverage run --branch --source=mmcv -m pytest tests/
+
+  build_cuda:
+    runs-on: ubuntu-latest
+    env:
+      CUDA: 10.1.105-1
+      CUDA_SHORT: 10.1
+      UBUNTU_VERSION: ubuntu1804
+      FORCE_CUDA: 1
+      MMCV_CUDA_ARGS: -gencode=arch=compute_61,code=sm_61
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
+        torch: [1.3.0, 1.5.0]
+        include:
+          - torch: 1.3.0
+            torchvision: 0.4.2
+          - torch: 1.5.0
+            torchvision: 0.6.0
+          - python-version: 3.8
+            torch: 1.5.0
+            torchvision: 0.6.0
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install CUDA
+        run: |
+          export INSTALLER=cuda-repo-${UBUNTU_VERSION}_${CUDA}_amd64.deb
+          wget http://developer.download.nvidia.com/compute/cuda/repos/${UBUNTU_VERSION}/x86_64/${INSTALLER}
+          sudo dpkg -i ${INSTALLER}
+          wget https://developer.download.nvidia.com/compute/cuda/repos/${UBUNTU_VERSION}/x86_64/7fa2af80.pub
+          sudo apt-key add 7fa2af80.pub
+          sudo apt update -qq
+          sudo apt install -y cuda-${CUDA_SHORT/./-} cuda-cufft-dev-${CUDA_SHORT/./-}
+          sudo apt clean
+          export CUDA_HOME=/usr/local/cuda-${CUDA_SHORT}
+          export LD_LIBRARY_PATH=${CUDA_HOME}/lib64:${CUDA_HOME}/include:${LD_LIBRARY_PATH}
+          export PATH=${CUDA_HOME}/bin:${PATH}
+          sudo apt-get install -y ninja-build
+      - name: Install Pillow
+        run: pip install Pillow==6.2.2
+        if: ${{matrix.torchvision == '0.4.2'}}
+      - name: Install PyTorch
+        run: pip install torch==${{matrix.torch}} torchvision==${{matrix.torchvision}}
       - name: Install system dependencies
         run: sudo apt-get update && sudo apt-get install -y ffmpeg libturbojpeg
       - name: Install unittest dependencies
+        run: pip install pytest coverage lmdb PyTurboJPEG
+      - name: Build and install
+        run: rm -rf .eggs && pip install -e .
+      - name: Run unittests
+        run: coverage run --branch --source=mmcv -m pytest tests/
+      # Only generate and upload coverage report for python3.7 && pytorch1.5
+      - name: Generate coverage report
```

Comment:
This step can be performed by each env, otherwise we can only see the coverage in a specific run.